src/services/jobs.js

```diff
@@ -0,0 +1,81 @@
+/**
+ *
+ * @module jobs
+ */
+
+import http from './http';
+
+/**
+ * Get active jobs for current user
+ * @param {String} details The user's details
+ * @return {Promise.<String>} User's active jobs
+ */
+const getActiveJobsForUser = details => {
+  return http.post(`jobs/getJobs`, details);
+};
+
+/**
+ * Get saved shifts for current user
+ * @return {Promise.<String>} User's active jobs
+ */
+const getSavedShiftsForUser = () => {
+  return http.get(`jobs/getSavedShifts/`);
+};
+
+/**
+ * Get active jobs for current user
+ * @param {Number} eml we don't know what this means yet
+ * @param {DateTime} shiftStart The start time of the shift
+ * @param {DateTime} shiftEnd The end time of the shift
+ * @param {Number} hoursWorked The number of hours worked
+ * @param {String} shiftNotes Shift notes
+ * @param {String} lastChangedBy The person who last updated the shift
```

Comment:
Should the lastChangedBy parameter be listed here? It is not actually in the declaration.

Reply:
I thought we got rid of that parameter, maybe git merge weirdness put it back. We'll look into it

Reply:
Update: evidently this one just slipped through our fingers and Travis passed it anyway. I'll get rid of it